src/main/java/net/tiagofar78/prisonescape/game/PrisonEscapeGame.java

```diff
@@ -62,8 +64,10 @@ public int playerJoin(Player player) {
 		if (isPlayerOnGame(playerName)) {
 			return -1;
 		}
-		
-		// TODO check if already started -2
+
+		if (_phase instanceof Ongoing) {
```

Comment:
Since we are using State design pattern you should be making booleans inside the phase class. In this case you should add a hasGameStarted method in Phase and make it return false in Waiting and true in Ongoing and Finished phases.